### PR TITLE
Fix error building Docker image of the initContainer

### PR DESCRIPTION
**Pull request by geoand**
Fixes: #9

### CODE REVIEW

File: supervisord/Dockerfile [hunk 1]
```diff
@@ -22,7 +22,7 @@ RUN mkdir -p ${SUPERVISORD_DIR}/conf ${SUPERVISORD_DIR}/bin
 
 COPY --from=builder /supervisord/bootstrap ${SUPERVISORD_DIR}/bin/
 COPY --from=builder /supervisord/execute.sh ${SUPERVISORD_DIR}/bin/
-COPY --from=builder /supervisord/conf/ ${SUPERVISORD_DIR}/
+COPY --from=builder /supervisord/conf/ ${SUPERVISORD_DIR}/conf/
```
Comment (cmoulliard) [inline supervisord/Dockerfile]: Does this behaviour has changed between different docker versions as I can build the image using the command `COPY --from=builder /supervisord/conf/ ${SUPERVISORD_DIR}/` ? @geoand

Comment (geoand) [reply supervisord/Dockerfile]: Very interesting :)

The truth is that I am using a very recent docker version (`18.03.1-ce`) which might account for the problem.
In that case I propose we leave this open and see if any other community members will be affected

**Comment by cmoulliard** [comment]
So. I will not merge the PR.

File: supervisord/Dockerfile (hunk 1 above)
Comment (cmoulliard) [reply supervisord/Dockerfile]: So. I will not merge the PR.

Comment (geoand) [reply supervisord/Dockerfile]: Sounds good. Let's just keep it open for the the time being

**Comment by geoand** [comment]
Let's close this

**Pull request closed without merging**